boards/FSM/Src/main.c

```diff
@@ -24,42 +24,6 @@
 
 /* Private includes ----------------------------------------------------------*/
 /* USER CODE BEGIN Includes */
-#include <assert.h>
-// shared
-#include "App_SharedMacros.h"
-#include "App_SharedStateMachine.h"
-#include "App_Timer.h"
-#include "App_CanAlerts.h"
-
-// IO functions exposing
-#include "Io_CanTx.h"
-#include "Io_CanRx.h"
-#include "Io_SharedSoftwareWatchdog.h"
-#include "Io_SharedCan.h"
-#include "Io_SharedHardFaultHandler.h"
-#include "Io_StackWaterMark.h"
-#include "Io_SoftwareWatchdog.h"
-#include "Io_Coolant.h"
-#include "Io_SharedHeartbeatMonitor.h"
-#include "Io_WheelSpeedSensors.h"
-#include "Io_SteeringAngleSensor.h"
-#include "Io_Adc.h"
-#include "Io_AcceleratorPedals.h"
-#include "Io_Brake.h"
-#include "Io_PrimaryScancon2RMHF.h"
-#include "Io_SecondaryScancon2RMHF.h"
-
-// world/state
-#include "App_FsmWorld.h"
-#include "states/App_DriveState.h"
-
-// Sensors
-#include "App_AcceleratorPedals.h"
-#include "App_Brake.h"
-#include "App_Coolant.h"
-#include "App_Steering.h"
-#include "App_Wheels.h"
-#include "configs/App_HeartbeatMonitorConfig.h"
```

Comment:
Uhhhh I don't think this was intentional, please undo all changes to FSM/Src/main.c